docs/concepts/overview/kubernetes-api.md

```diff
@@ -24,11 +24,11 @@ In our experience, any system that is successful needs to grow and change as new
 
 What constitutes a compatible change and how to change the API are detailed by the [API change document](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api_changes.md).
 
-## OpenAPI and Swagger definitions
+## OpenAPI definition
 
 Complete API details are documented using [Swagger v1.2](http://swagger.io/) and [OpenAPI](https://www.openapis.org/). The Kubernetes apiserver (aka "master") exposes an API that can be used to retrieve the Swagger v1.2 Kubernetes API spec located at `/swaggerapi`. You can also enable a UI to browse the API documentation at `/swagger-ui` by passing the `--enable-swagger-ui=true` flag to apiserver.
 
-We also host a version of the [latest v1.2 API documentation UI](http://kubernetes.io/kubernetes/third_party/swagger-ui/). This is updated with the latest release, so if you are using a different version of Kubernetes you will want to use the spec from your apiserver.
+We host a version of the [latest {{page.version}} API documentation UI](https://kubernetes.io/docs/api-reference/{{page.version}}/). This is updated with the latest release, so if you are using a different version of Kubernetes you will want to use the spec from your apiserver.
```

Comment:
This is updated with the latest release, so if you are using a different version of Kubernetes you will want to use the spec from your apiserver.

this no longer applies. You can just delete this paragraph as the /docs/api-reference is already shown on the top of Reference section.